kubernetes/charts/weblogic-operator/templates/_domain-namespaces.tpl

```diff
@@ -1,10 +1,17 @@
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 {{- define "operator.domainNamespaces" }}
+{{- if not .dedicated }}
 {{- $args := include "utils.cloneDictionary" . | fromYaml -}}
 {{- range $key := $args.domainNamespaces -}}
 {{-   $ignore := set $args "domainNamespace" $key -}}
 {{-   include "operator.operatorRoleBindingNamespace" $args -}}
 {{- end }}
+{{- else }}
+{{- $args := include "utils.cloneDictionary" . | fromYaml -}}
+{{- $key := .Release.Namespace -}}
```

Comment:
is this right? looks wrong

Reply:
If the dedicated is set to true, we ignore the domainNamespaces setting, and create the namespaced roles and rolebindings for the operator's namespace only.

Reply:
this works as expected.